linode/manager
# refactor: [M3-6265] - MUI v5 Migration - `SRC > Features > Firewalls`

## Description 📝
Migrates SRC > Features > Firewalls from MUI v4 to v5

## Major Changes 🔄
- Removed @mui/styles imports and converted most styling to styled components
- Removed React.FC
- removed default exports for functional components in favor of inline exports unless they were being lazy exported, then kept default exports
- renamed component props to be more specific, only for props that were exported
- removed index files that only had one export on them 

## How to test 🧪
- go to the /firewalls section locally and ensure all functionality works and that things look the same as prod
  - note: there seems to be a slight difference in width for the first table cell from the landing page (at least for my local), same as the discrepancy [here](https://github.com/linode/manager/pull/9403) on local compared to prod - likely isn't related to this specific PR
